bin/separate-app-worker-2

```diff
@@ -0,0 +1,22 @@
+#!/bin/bash
+
+set -eu
+
+ENVIRONMENT=$1
+
+# echo Start new worker
+# cf start "teaching-vacancies-worker-$ENVIRONMENT"
+
+# echo Start temp webapp
+# cf start "teaching-vacancies-$ENVIRONMENT-TEMP"
+
+echo Send traffic to temp webapp
+cf map-route "teaching-vacancies-$ENVIRONMENT-TEMP" london.cloudapps.digital --hostname "teaching-vacancies-$ENVIRONMENT"
+
+echo Delete old webapp
+cf delete "teaching-vacancies-$ENVIRONMENT" -f
+
+echo Deploy new webapp
+cf push -f "paas/web/manifest-$ENVIRONMENT.yml" --no-start teaching-vacancies-$ENVIRONMENT
```

Comment:
you need to rename `paas/web/manifest.yml` and `paas/worker/manifest.yml` to `paas/web/manifest-production.yml` and `paas/worker/manifest-production.yml`